### DISCUSSION TITLE
1 test failing while running an example

### DISCUSSION THREAD

**Discussion by hardud** [idaholab/moose#26539] [Q&A Getting Started]
While running an example with Pre-built moose, most tests are getting passed, a few are being skipped and "1 test is always failing" with whatever example I run. 

Also, under the "ls $CONDA_PREFIX/moose/share/moose", I am unable to find 'heat_conduction'

I therefore tried running 'heat_transfer' and 'porous_flow' but even for them, 1 test is failing every time.

Does anyone have any idea what's happening?

**Comment by GiudGiud** [comment]
Hello

For heat_conduction, that's the former name of the heat_transfer module. No need to try to run the heat_conduction tests.

Guillaume
